src/doc/trpl/match.md

```diff
@@ -20,25 +20,26 @@ match x {
 }
 ```
 
-`match` takes an expression and then branches based on its value. Each *arm* of
-the branch is of the form `val => expression`. When the value matches, that arm's
+`match` takes an expression and then branches based on its value. Each branch or
+*case* is of the form `val => expression`. When the value matches, that case's
```

Comment:
RBE was actually entirely not written by me at all :) Just adopted.

I'm not sure where it comes from historically, but I vaguely remember using 'arms' to describe pattern matching in Haskell, too. They've been called arms as long as I can remember, which is the primary justification, I guess. Maybe an MLism?